0.download_data/2.download_images.py

```diff
@@ -157,10 +157,13 @@ def download_jump_cpg000_images_from_s3(
     return "data/images"
 
 
+# -
+
+
```

Comment:
Is this supposed to be blank?

Reply:
Thanks - no, I'll remove from the notebook!